backuprestore/backup.py

```diff
@@ -41,7 +41,7 @@ def main():
                 iaas_client.exit('Could not find the snapshot of the persistent volume {}.'
                                  .format(DIRECTORY_PERSISTENT))
 
-            if landscape != 'Aws' and landscape != 'Azure':
+            if landscape != 'Aws' and landscape != 'Azure' and landscape != 'Gcp':
```

Comment:
```
if landscape == 'Openstack'
```
3 and-ed string comparison . Can we just do one ?

Reply:
I think this was done as the default way of backup was via data tarball upload approach and only iaas having different approach had its own way which are below conditions ```if landscape == 'Azure' or landscape == 'Gcp':```
If everyone votes for this change, I will change the condition to ```if landscape == 'Openstack'```

Reply:
we can change it to `if landscape == 'Openstack'`

Reply:
Ok.. For simplicity we can do following:
1. create a list of IaaS supports snapshots. Let's **snapshot_supported_iaas=['Aws', 'Azure', 'Gcp']**;
    or **blob_supported_iaas=['Openstack']**
2. check if landscape in snapshot_supported_iaas/blob_supported_iaas